src/components/cards/WarningCard.tsx

```diff
@@ -39,7 +39,7 @@ export function WarningCard({ title, header, points, footer, marginRem, paddingR
 
   const renderBulletpoint = (message: string, index: number) => {
     return (
-      <View style={styles.bulletpointRow} key={index}>
+      <View style={styles.bulletpointRow} key={`${index}: ${message}`}>
```

Comment:
Goal is to avoid using the index at all. The key should unique and stable and array indexes are not stable. In this particular use case it is stable but it's an anti-pattern and we should avoid doing it.